e2e/basics.spec.js

```diff
@@ -323,81 +323,81 @@ describe('Basic React Native and Interaction Support', () => {
     it("should autotrack 'TouchableOpacity's", async () => {
       const expectedHierarchy = `${BASICS_PAGE_TOP_HIERARCHY}TouchableOpacity;[testID=touchableOpacityText];|`;
       const expectedTargetText = 'Touchable Opacity Foo';
-      await rnTestUtil.assertAutotrackHierarchy('touchableHandlePress', {
-        touchableHierarchy: expectedHierarchy,
-        targetText: expectedTargetText,
-        screenName: 'Basics',
+      await rnTestUtil.assertAutotrackHierarchy('touch', {
+        hierarchy: expectedHierarchy,
+        target_text: expectedTargetText,
+        screen_name: 'Basics',
         path: 'Basics',
       });
     });
 
     it("should autotrack 'TouchableHighlight's", async () => {
       const expectedHierarchy = `${BASICS_PAGE_TOP_HIERARCHY}TouchableHighlight;[testID=touchableHighlightText];|`;
       const expectedTargetText = 'Touchable Highlight';
-      await rnTestUtil.assertAutotrackHierarchy('touchableHandlePress', {
-        touchableHierarchy: expectedHierarchy,
-        targetText: expectedTargetText,
-        screenName: 'Basics',
+      await rnTestUtil.assertAutotrackHierarchy('touch', {
+        hierarchy: expectedHierarchy,
```

Comment:
Is hierarchy, without any sort of qualifier, going to cause any potential confusion anywhere? react_hierarchy?

Reply:
This will be the only hierarchy coming from the `react_native` lib, so I wouldn't think so, but I'll defer to you

Reply:
If you don't see any concern, then I don't see any concern. Thanks!